CDP4Addin/Addin.cs

```diff
@@ -109,17 +111,46 @@ public Addin()
         {
             logger.Debug("starting CDP4-CE addin");
 
+            var appSettings = new AppSettingsService();
```

Comment:
You have implemented the AppSettingsService as an IAppSettingsService with all the MEF whistles and bells. Please make use of Inversion of Control and initialize the service in the `InitializeMefImports` method

Reply:
agreed